aws/sdk/examples/apigateway/src/bin/get_rest_apis.rs

```diff
@@ -49,22 +71,15 @@ async fn main() -> Result<(), Error> {
     let shared_config = aws_config::from_env().region(region_provider).load().await;
     let client = Client::new(&shared_config);
 
-    let resp = client.get_rest_apis().send().await?;
-
-    for api in resp.items.unwrap_or_default() {
-        println!("ID:          {}", api.id.as_deref().unwrap_or_default());
-        println!("Name:        {}", api.name.as_deref().unwrap_or_default());
-        println!(
-            "Description: {}",
-            api.description.as_deref().unwrap_or_default()
-        );
-        println!(
-            "Version:     {}",
-            api.version.as_deref().unwrap_or_default()
-        );
-        println!("Created:     {}", api.created_date.unwrap().to_chrono());
-        println!();
-    }
+    show_apis(&client).await.unwrap();
 
     Ok(())
 }
+
+#[actix_rt::test]
+async fn test_get_rest_apis() {
+    let shared_config = aws_config::load_from_env().await;
+    let client = Client::new(&shared_config);
+
+    client.get_rest_apis();
```

Comment:
That said, I'm not sure this test is giving us any benefit. This will only construct a new builder for this operation on the client, and this is thoroughly tested throughout the SDK's automation.